README.md

```diff
@@ -6,7 +6,7 @@ The Feature Browser tool generates .html files to represent your .feature files
 ## Installation
 Install via composer/packagist:
 ```
-composer require "feature-browser/feature-browser": "~1.0"
+composer require --dev "feature-browser/feature-browser": "1.0.0"
```

Comment:
Personally I think leave the `--dev` bit out, leave it up to developer discretion (you could add a note about it I guess?). I also think `~1.0` is better than `1.0.0` or just omit the version number altogether and let composer work it out